### PR TITLE
#30: Completion. Remove PHP completions from all not appropriate places in XML files

### PR DESCRIPTION
**Pull request by konarshankar07**
### Description (*)

This PR will fix the PHP class and module name completion from inappropriate places 
### Fixed Issues (if relevant)

1. magento/magento2-phpstorm-plugin#30: Completion. Remove PHP completions from all not appropriate places in XML files

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages

### CODE REVIEW

File: src/com/magento/idea/magento2plugin/completion/xml/XmlCompletionContributor.java [hunk 1]
```diff
@@ -18,22 +21,46 @@
...
-                new PhpClassMemberCompletionProvider(),
-                new ModuleNameCompletionProvider(),
+        extend(CompletionType.BASIC, psiElement(XmlTokenType.XML_ATTRIBUTE_VALUE_TOKEN)
+                        .inside(XmlPatterns.xmlAttribute().withName(CommonXml.ATTR_CLASS)),
+                new PhpClassCompletionProvider()
+        );
+
+        /* File Path Completion provider */
+        extend(CompletionType.BASIC, psiElement(XmlTokenType.XML_ATTRIBUTE_VALUE_TOKEN)
+                        .inside(XmlPatterns.xmlAttribute().withName("template")),
```
Comment (VitaliyBoyko) [inline src/com/magento/idea/magento2plugin/completion/xml/XmlCompletionContributor.java:62]: Hi.
I've refactored most of the string literals to constant classes. Please do the same way.
 Thank you!

Comment (VitaliyBoyko) [reply src/com/magento/idea/magento2plugin/completion/xml/XmlCompletionContributor.java:62]: We are storing all magento related string literals under `com/magento/idea/magento2plugin/magento`.

Comment (VitaliyBoyko) [reply src/com/magento/idea/magento2plugin/completion/xml/XmlCompletionContributor.java:62]: I'm about the new changes.

Comment (konarshankar07) [reply src/com/magento/idea/magento2plugin/completion/xml/XmlCompletionContributor.java:62]: Okay, I will make sure of these changes

**Comment by VitaliyBoyko** [comment]
I see other issues, however, let's fix them in the scope of a separate task.

**Comment by VitaliyBoyko** [comment]
This PR includes reward labels for #62.
The Reward system was not configured for March, so we adjusted points for April.

**Pull request merged**